tests/Unit/Helpers/Evolution/Systems/Cce/VolumeTestHelpers.hpp

```diff
@@ -116,8 +116,18 @@ using matching_variables_tag = ::Tags::Variables<
 using volume_spin_weighted_variables_tag = ::Tags::Variables<
     tmpl::list<Tags::BondiJ, Tags::Dy<Tags::BondiJ>, Tags::BondiJCauchyView,
                Tags::Dy<Tags::BondiJCauchyView>,
+               Tags::Dy<Tags::Dy<Tags::BondiJ>>,
                Tags::Dy<Tags::Dy<Tags::BondiJCauchyView>>, Tags::Psi0Match,
-               Tags::OneMinusY, Tags::Dy<Tags::Psi0Match>, Tags::BondiU>>;
+               Tags::OneMinusY, Tags::Dy<Tags::Psi0Match>, Tags::BondiU,
```

Comment:
I think these changes can be dropped now since you're not modifying the volume test?

Reply:
Ping on this

Reply:
Done!